src/geometry/mrob/SE3vel.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef SE3VEL_HPP_
+#define SE3VEL_HPP_
+
+#include <iostream>
+
+#include "mrob/matrix_base.hpp"
+#include "mrob/SO3.hpp"
+
+namespace mrob{
+
```

Comment:
here requires a brief comment on the class, what kind of 5x5 structure is in here